src/main/java/core/basesyntax/FruitInventory.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitInventory {
+
+    private final String fruit;
+    private final int quantity;
+
+    private Map<String, Integer> inventory;
```

Comment:
for what we need 
private quantity, fruit

and then map?

As I see, we can leave only map.